how-to-guides/re-run-failed-training/scripts/re_run_failed_training.py

```diff
@@ -0,0 +1,113 @@
+import neptune.new as neptune
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import os
+from torchvision import datasets, transforms
+
+# Step 1: Get Run ID
+
+# Fetch project
+project = neptune.get_project(name='common/showroom', api_token='ANONYMOUS')
```

Comment:
Please make sure that you use `"` not `'` everywhere in the code -> proper code formatting -> use black